src/services/sanity.js

```diff
@@ -2278,3 +2278,13 @@ export async function fetchShows(brand, type, sort = 'sort') {
   })
   return fetchSanity(query, true)
 }
+
+export async function fetchMethodV2IntroVideo(brand) {
+  const _type = 'method-intro'
+  const filter = `_type == '${_type}' && brand == '${brand}'`;
+
+  const finalFilter = await new FilterBuilder(filter).buildFilter()
+  const query =`*[${finalFilter}]`
```

Comment:
Is this expected to be restricted based on permission + status? I did not see either in the spec doc.


We need projections for the object fields.

something  like below:
```suggestion

  const query =`*[${filter}] {
    title,
    brand,
    'description': text::pt(description),
    'thumbnail': thumbnail.asset->url,
    'slug': slug.current,
    video_mobile,
    video_desktop,
  }`
```